pkg/table/table.go

```diff
@@ -243,6 +255,17 @@ type RemoveRecordOption interface {
 	applyRemoveRecordOpt(*RemoveRecordOpt)
 }
 
+// ExtraPartialRowOption is the combined one of IndexesLayout and ColumnSizeOption.
+type ExtraPartialRowOption struct {
+	IndexesRowLayout  IndexesLayout
+	ColumnsSizeHelper *ColumnsSizeHelper
+}
```

Comment:
`ExtraPartialRowOption` and `RemoveRecordOpt` have exactly the same definition. Why don't we just pass a `RemoveRecordOpt` struct.

Reply:
This one is to be consistent with @lcwangchao 's new API changes